### ISSUE TITLE
Drone operator should be able to adjust Flight plan before starting flight.

### ISSUE THREAD

**Issue by aadarshdhakal** [hotosm/drone-tm#51]
**As a Drone Operator,** I must be able to adjust the orientation of flight and flying height of the drone by few meters up to the limit by which the GSD doesn't get effected.

**Comment by spwoodcock** [comment]
Sorry didn't see this! Closing in favour of #562
